src/rot_covariances.jl

```diff
@@ -0,0 +1,134 @@
+@doc """
+
+Function to convert a covariance matrix containing one rotation parameterization to another:
+
+Inputs:
+
+    tfunc     - transformation function, should take a Vector with length size(cX, 1) as an input and return the transformed state vector
+
+    cX        - covariance matrix 
+
+    xbar      - mean X in the input space (set = nothing to enforce zero mean in input and output)
+
+    oT_cov    - output type for the covariance matrix (default Matrix)
+
+    oT_mean   - output type for the covariance matrix (default Vector)
```

Comment:
This is a homeless bit of code for applying non-linear transforms to distributions.  Its living here because the non-linear transforms I'm using it for are conversions between rotation parametrization.  Its a nasty bit of code because the output type (dimensionality) of tfunc is unknown making it crappy with fixed size stuff.

I'd vote for moving it to transforms style package that stores input / output dims along with the transform, if you've heard of one?